Huion_H640P/descriptors.txt

```diff
@@ -0,0 +1,90 @@
+
+Bus 003 Device 003: ID 256c:006e  
+Device Descriptor:
+  bLength                18
+  bDescriptorType         1
+  bcdUSB               1.10
+  bDeviceClass            0 (Defined at Interface level)
+  bDeviceSubClass         0 
+  bDeviceProtocol         0 
+  bMaxPacketSize0         8
+  idVendor           0x256c 
+  idProduct          0x006e 
+  bcdDevice            0.00
+  iManufacturer           5 (error)
+  iProduct                6 Tablet Monitor
```

Comment:
Really, a "Tablet Monitor"? Still not as bad as misspelling the company name I saw once :)